packages/core/util/functionStrings.ts

```diff
@@ -1,4 +1,5 @@
 export const functionRegexp = /^\s*function\s*\w*\s*\(([^)]*)\)\s*{([\w\W]*)/
+export const anywhereFunctionRegexp = /\s*function\s*\w*\s*\(([^)]*)\)\s*{([\w\W]*)/
```

Comment:
I'd like to not have to worry about keeping both regexps in sync, I can see one getting updated without the other accidentally. Maybe in your file you could import the existing `functionRegexp` and then modify it (maybe using `functionRegexp.toString()`, then removing the caret, and using `new RegExp(functionRegexpString)`).

Reply:
got it, doing the following now:
const anywhereFunctionRegexp = new RegExp(functionRegexp.toString().slice(2))